scripts/copy-generated.js

```
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+/**
+ * Copy generated files script
+ *
+ * This script copies the generated files to the dist directory
+ */
+
+const fs = require("fs");
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)